capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
+end
```

Comment:
- [ ] The capitalize decorator suppose to capitalize the first character of the name not making the whole name in uppercase, so please fix that.
[Hint] you can do it like that:
```ruby
 def correct_name
    @nameable.correct_name.capitalize
  end
```

Reply:
Hello @Omar-Muhamad, thank you for the feedback. I will do it immediately.